jovian/jovian_nb_ext/main.js

```diff
@@ -741,19 +735,27 @@ define([
     jvn_notif.element.attr("disabled", true);
   }
 
-  function dropdownOption(modal, option) {
+  function openModal(func, modal = null) {
```

Comment:
This helper function is the only way that I know to solve the keyboard loss issue. I have tested on both linux and mac, it works perfectly. If there are any option, please let me know, I will change to solve this keyboard loss issue.

Reply:
Okay you can look at `validateapi` function for refrence. We can come back to this after the launch.